style.css

```
@@ -42,4 +42,54 @@ h1 {
 h2{
     line-height: 0;
     font-size: 1.3em;
-}
\ No newline at end of file
+}
+
+table {
+  width: 100%;
+  table-layout: fixed;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+th, td {
+  width: 33.33%; 
+  text-align: left;
+  position: relative;
+  padding: 10px; 
+}
+
+th {
+  border-bottom: 3px solid black; 
+  font-weight: bold; 
+  background-color: #d3d3d3;
+}
+
+td {
+  border-bottom: 1px solid #ccc;
+  padding-right: 30px;
+}
+
+button.delete-button {
+  position: absolute; 
+  top: 5px; 
+  right: 5px;
```

```suggestion
  right: 0;
```

this also, can you set this to 0? These two changes with added padding will make sure the text and the delete button don't overlap